homeassistant/components/tasmota/sensor.py

```diff
@@ -63,28 +53,54 @@
 ICON = "icon"
 
 # A Tasmota sensor type may be mapped to either a device class or an icon, not both
-SENSOR_DEVICE_CLASS_ICON_MAP = {
-    hc.SENSOR_AMBIENT: {DEVICE_CLASS: DEVICE_CLASS_ILLUMINANCE},
-    hc.SENSOR_APPARENT_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
+SENSOR_DEVICE_CLASS_ICON_MAP: dict[str, dict[str, Any]] = {
+    hc.SENSOR_AMBIENT: {
+        DEVICE_CLASS: SensorDeviceClass.ILLUMINANCE,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
+    hc.SENSOR_APPARENT_POWERUSAGE: {
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
```

Comment:
You seem to have lost the DeviceClass here.
```suggestion
        DEVICE_CLASS: SensorDeviceClass.POWER
        STATE_CLASS: SensorStateClass.MEASUREMENT,
```

Reply:
That's intentional, is not correct to set a sensor reporting apparent power as device class power